boards.txt

```diff
@@ -21180,7 +21180,6 @@ heltec_wireless_stick.build.bootloader_addr=0x1000
 heltec_wireless_stick.build.target=esp32
 heltec_wireless_stick.build.mcu=esp32
 heltec_wireless_stick.build.core=esp32
-heltec_wireless_stick.build.variant=heltec_wireless_stick
```

Comment:
Why not keeping this, as the variant is the same for all both revisions?

Reply:
Changed - I thought it were required for any revision.